examples/jsm/loaders/SVGLoader.js

```diff
@@ -398,15 +398,25 @@ SVGLoader.prototype = Object.assign( Object.create( Loader.prototype ), {
 						var numbers = parseFloats( data );
 
 						for ( var j = 0, jl = numbers.length; j < jl; j += 7 ) {
+							// skip command if start point == end point
+							if( numbers[ j + 5 ] == point.x && numbers[ j + 6 ] == point.y ) continue
 
+							
 							var start = point.clone();
 							point.x = numbers[ j + 5 ];
 							point.y = numbers[ j + 6 ];
 							control.x = point.x;
 							control.y = point.y;
-							parseArcCommand(
-								path, numbers[ j ], numbers[ j + 1 ], numbers[ j + 2 ], numbers[ j + 3 ], numbers[ j + 4 ], start, point
-							);
+
+							if( numbers[ j ] == 0 || numbers[ j + 1 ] == 0 ) {
```

Comment:
I think it is better to move this 'if' into the ```parseArcCommand``` function, since it is repeated for the 'a' and 'A' commands.

Reply:
made the changes [here](https://github.com/mrdoob/three.js/pull/20644/commits/66b7db927b6947c629c3cf9b266824a8474c5898#r519479351)